crt_portal/cts_forms/views.py

```diff
@@ -182,6 +183,38 @@ def _related_reports_count(report):
         return 1
 
 
+def _get_email(report):
+    return report.contact_email
+
+
+def _set_constant_writer(contact_templates, report):
+    for contact_template in contact_templates:
+        if contact_template[1].upper() == report.contact_email.upper():
+            report.constant_writer = True
```

Comment:
I'm not sure how I feel about setting this directly on the report, since the value isn't persistent and shouldn't be saved. Especially in this case, as this field has some history

What do you think about:
- Returning a `dict` of report_ids instead of setting values on the report itself
- Using the letter titles/etc (rather than a boolean)

Reply:
@kewitham @okkays we may need some place to save 'constant writer' for the future export and for analytics. We probably need to choose a place that can scale for other tags later on e.g. 'repeated email'